### ISSUE TITLE
Hodnocení

### ISSUE THREAD

**Issue by LenkaSilna** [danka0ro/PROJEKT-Vizitka#1]
Hodnotím jako výborné zpracování. Možná bych se více zamyslela nad pojmenováním, tady tříd odstavců, obecně pojmenování je jedna z nejtěžších disciplín v programování :D. Dobrá práce
Ještě jedna na závěr, všimla jsem si toho až po kontrole jinde. Text do bloku má být v obou odstavcích, ten první do okrajů není. Zkusíš opravit?

**Comment by danka0ro** [comment]
Nastavila jsem vlastnost u odstavců na `text-align: justify;`. Je to tak nyní správně? Ale slova mi nevyšla na ten samý řádek jako je tomu v zadání...

**Comment by LenkaSilna** [comment]
Ano, ale existuje vlastnost, která upravuje rozestup mezi písmeny, česky se tomu říká prostrkání. Zkus ji najit a nastavit tak aby slova odpovídala.

**Comment by danka0ro** [comment]
`letter-spacing:` nastaven na `-0.1px;` a slova odpovídají :-) jupí. Děkuji.
